interfaces/Portalicious/src/app/domains/project/project.helper.ts

```diff
@@ -12,15 +16,76 @@ import {
   AttributeWithTranslatedLabel,
   Project,
 } from '~/domains/project/project.model';
+import { LANGUAGE_ISO_639_1 } from '~/domains/registration/registration.helper';
 
-export const ATTRIBUTE_LABELS: Record<string, string | undefined> = {
-  fspDisplayName: $localize`:@@attribute-label-fspDisplayName:FSP Display Name`,
+export const ATTRIBUTE_LABELS: Record<GenericRegistrationAttributes, string> = {
```

Comment:
This file is getting a bit crowded. I wonder if we should have a dedicated `attribute` domain / helper - but not sure if that makes also from a BE perspective.

Reply:
After a discussion with Peter and Elwin we decided to do 

domains/project/project-attribute.helper.ts